ui/app/services/store.js

```diff
@@ -9,9 +9,10 @@ import { copy } from 'ember-copy';
 import { resolve, Promise } from 'rsvp';
 import { dasherize } from '@ember/string';
 import { assert } from '@ember/debug';
-import { set, get, computed } from '@ember/object';
+import { set, get } from '@ember/object';
 import clamp from 'vault/utils/clamp';
 import config from 'vault/config/environment';
+import { tracked } from 'tracked-built-ins/.';
```

Comment:
Not sure if this is a typo! Why is tracked-built-ins being imported this way `import { tracked } from 'tracked-built-ins/.';`? Could we import it like this `import { tracked } from 'tracked-built-ins'`?

Reply:
Good catch! VSCode automatically added this import, not sure why it did it that way...